live-examples/js-examples/intl/meta.json

```diff
@@ -239,6 +239,12 @@
             "fileName": "intl-supportedvaluesof.html",
             "title": "JavaScript Demo: Intl.supportedValuesOf",
             "type": "js"
+        },
+        "intlPluralRulesPrototypeResolvedOptions": {
+            "exampleCode": "./live-examples/js-examples/intl/intl-pluralrules-prototype-resolvedoptions.js",
+            "fileName": "intl-pluralrules-prototype-resolvedoptions.html",
+            "title": "JavaScript Demo: Intl.PluralRules.prototype.resolvedOptions()",
+            "type": "js"
```

Comment:
This needs to be inserted into the object in alphabetical order, to make it easier for maintainers to find it.